plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -189,11 +189,10 @@ function ListMenuItem:update()
     }
 
     local function _fontSize(nominal)
---        return math.ceil(nominal * font_scale_factor * scale_by_size
-        -- dimen.h * scaleBySize, font_size / scaleBySize cancel each other out
-        local font_size = math.ceil(nominal * dimen.h / 64)
-        logger.warn("font size", nominal, dimen.h, font_size)
-        return font_size
+        -- nominal font size is based on 64px ListMenuItem height
+        -- keep ratio of font size to item height
+        local font_size = nominal * dimen.h / 64 / scale_by_size
```

Comment:
OK :) (On the emulator, I didn't realized it was missing the scaleBySize inversion :)

Reply:
That's because scaleBySize in the emulator (720/540) has no effect. On 1200/900 the difference would be more than 10%.